contrib/dockerfiles/aarch64-linux-gnu.dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG TARGET=aarch64-linux-gnu
 
 # -----------
-FROM ubuntu:latest as builder
+FROM --platform=linux/amd64 ubuntu:latest as builder
```

Comment:
Question: Do the builders have to be platform specific now? Since the previous issue was dev on apple arm systems triggered a mix of x64 and arm. The aarch64 support on apple platform now picks the right default. Does this not work better without?

Reply:
Noted on this. I will consolidate the various dockerfiles and clean up our build process in a separate PR in the near future.